# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/cocoa/GfxInfo.h
# Commit: c14528f13392
# Full Hash: c14528f133927f0b5f4636d8d85ce7fb8b822dbc
# Author: Jeff Muizelaar <jmuizelaar@mozilla.com>
# Date: 2020-09-28 21:28:24
# Description:
#   Bug 1664843. Revert "Bug 1649490 - detect all Mac GPUs" r=miko
#   
#   This reverts commit f559b920572b19d1f943744416c38baf82780b98
#   for causing crashes on Google maps on Intel HD 3000 hardware
#   
# ==============================================================================

diff -r 85a37b51bf2e -r c14528f13392 widget/cocoa/GfxInfo.h
--- a/widget/cocoa/GfxInfo.h	Mon Sep 28 12:47:31 2020 +0000
+++ b/widget/cocoa/GfxInfo.h	Mon Sep 28 13:00:11 2020 +0000
@@ -81,16 +81,14 @@
   void GetSelectedCityInfo();
   void AddCrashReportAnnotations();
 
-  uint32_t mNumGPUsDetected;
+  uint32_t mAdapterRAM;
+  nsString mDeviceID;
+  nsString mDriverVersion;
+  nsString mDriverDate;
+  nsString mDeviceKey;
 
-  uint32_t mAdapterRAM[2];
-  nsString mDeviceID[2];
-  nsString mDriverVersion[2];
-  nsString mDriverDate[2];
-  nsString mDeviceKey[2];
-
-  nsString mAdapterVendorID[2];
-  nsString mAdapterDeviceID[2];
+  nsString mAdapterVendorID;
+  nsString mAdapterDeviceID;
 
   uint32_t mOSXVersion;
 };